pandas/tests/base/test_ops.py

```diff
@@ -137,227 +137,233 @@ def setup_method(self, method):
         self.is_valid_objs = self.objs
         self.not_valid_objs = []
 
-    def test_none_comparison(self):
+    def test_none_comparison(self, series_with_differing_index):
```

Comment:
series_with_differing_index makes me think "differing from what"?  i think Will had a similar issue

Reply:
What about `series_with_changing_index`?

Reply:
`f8series_any_simple_index`?

Reply:
What does the f8 stand for?

Reply:
float64 dtype

Reply:
Ok, I'll go with it then